.github/workflows/tests.yml

```diff
@@ -31,3 +31,28 @@ jobs:
         playwright install chromium --with-deps
     - name: Run tests
       run: tox
+
+  # Verify that docs build
+  test_docs:
```

Comment:
Added a test to check that the docs do build, to avoid accidentlly breaking them again.